wiki0/src/server/redis/client.ts

```diff
@@ -0,0 +1,12 @@
+import { createClient } from 'redis';
+
+const createRedisClient = async () => {
```

Comment:
a redis client is already created in wiki0/src/server/main.ts.  You can thread it through the express app.  See https://expressjs.com/en/4x/api.html#app.locals